tensorrt_llm/serve/openai_server.py

```
@@ -324,6 +340,68 @@ async def create_chat_response(
+                model=self.model,
+                choices=[ChatCompletionResponseChoice(index=0, message=ChatMessage(role="assistant", content="dummy"), mm_embedding_handle=mm_embedding_handle, finish_reason="length")],
+                usage=UsageInfo(prompt_tokens=num_tokens, completion_tokens=1, total_tokens=num_tokens+1))
+
+        try:
+            check_multiple_response(request.n, self.llm.args.backend)
+            conversation: List[ConversationMessage] = []
+            tool_dicts = None if request.tools is None else [
+                tool.model_dump() for tool in request.tools
+            ]
+            # TODO: placeholder for multimodal disagg e2e
+            disaggregated_params = to_llm_disaggregated_params(request.disaggregated_params)
```

Ruff (0.12.2)

Local variable `disaggregated_params` is assigned to but never used

Remove assignment to unused variable `disaggregated_params`

(F841)